env.js

```diff
@@ -113,6 +113,7 @@ function setStorage() {
   env.profile_collection = readENV('MONGO_PROFILE_COLLECTION', 'profile');
   env.devicestatus_collection = readENV('MONGO_DEVICESTATUS_COLLECTION', 'devicestatus');
   env.food_collection = readENV('MONGO_FOOD_COLLECTION', 'food');
+  env.azurepush_collection = readENV('MONGO_AZUREPUSH_COLLECTION', 'azurepush');
```

Comment:
Right now, this collection stores device registrations, not notifications themselves.  The concept is similar to what we have in static environment variables for pushover, but they are stored in mongo since device registration is dynamic.

I can see two approaches to this:
1. Have a single collection for `notifications` that includes at it's top level devices and settings (what I currently store) and then has an array of actual notifications sent/receipts.
2. Use two separate collections: one for `devices` (what I currently store) and another for `notifications` which can have a key to the devices it was sent to/received by.

Thoughts?